### ISSUE TITLE
Render and define custom emoji/reactions

### ISSUE THREAD

**Issue by ajkessel** [mszpro/Nil#11]
Matrix now has a spec for custom emojis that can be used in reactions. Some clients have limited support. Would be great if Nil could at least render mxc:// urls in reactions.

https://github.com/matrix-org/matrix-spec-proposals/pull/4027

**Comment by mszpro** [comment]
Good idea! I think I can implement a lot to detect Matrix media URL and show them inline with the text.
